.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,15 @@
-*Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots.*
+<!--
+
+    Add a concise description of what this PR is changing or adding, and why. Consider including before/after screenshots.
+    Consider mentioninig issues related to this pull request
+
+-->
 
 ## Pre-launch Checklist
 
-- [ ] I have run `dartfmt` on all changed files
+- [ ] I have run `dartfmt` on all changed files <!-- THIS IS REQUIRED -->
+- [ ] I have incremented the package version as appropriate and updated `CHANGELOG.md` with my changes <!-- THIS IS REQUIRED -->
+- [ ] I have added/updated relevant documentation <!-- THIS IS REQUIRED -->
 - [ ] I have run "optimize/organize imports" on all changed files
 - [ ] I have addressed all analyzer warnings as best I could
-- [ ] I have added/updated relevant documentation
-- [ ] I have incremented the package version as appropriate and updated `CHANGELOG.md` with my changes
 - [ ] I have run `flutter pub publish --dry-run` and addressed any warnings
```

Comment:
Maybe we should make this a maintainer-only thing. So only visible for PR's to `stable`. I guess we'd need a different template entirely for that?